pkgs/by-name/he/hexpatch/package.nix

```
@@ -0,0 +1,52 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  cmake,
+  python3,
+  stdenv,
+  gcc,
+  llvmPackages,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "hexpatch";
+  version = "1.9.3";
+
+  src = fetchFromGitHub {
+    owner = "Etto48";
+    repo = "HexPatch";
+    rev = "v${version}";
+    hash = "sha256-yyYxRJ+o+Z5z7PmjcFCsahRXZ9JHFmGmituzGTxY6ec=";
+  };
+
+  cargoHash = "sha256-bckZtHzgKQow7l2Cg6Xi1oNk13FH2xKe8cbZ/Xj9E04=";
+
+  nativeBuildInputs =
+    [
+      cmake
+      python3
+    ]
+    ++ lib.optionals stdenv.hostPlatform.isLinux [ gcc ]
+    ++ lib.optionals (stdenv.hostPlatform.isWindows || stdenv.hostPlatform.isDarwin) [
+      llvmPackages.libclang
+    ];
+
+  postFixup = ''
+    ln -s $out/bin/hex-patch $out/bin/hexpatch
+  '';
+
+  meta = {
+    description = "A binary patcher and editor written in Rust with terminal user interface.";
```

We try to avoid an indefinite article in descriptions.

See https://github.com/NixOS/nixpkgs/tree/00cd61f517aa325478e9f743d572f65d2bbcb16a/pkgs#meta-attributes and/or https://github.com/NixOS/nixpkgs/pull/317959

```suggestion
    description = "Binary patcher and editor written in Rust with terminal user interface.";
```